.openshift/templates/ninja-board-build.yml

```diff
@@ -37,10 +37,14 @@ objects:
         FROM openshift3/jboss-webserver31-tomcat7-openshift:latest
         USER 0
         RUN yum clean all && \
-          INSTALL_PKGS="python-requests pytz" && \
+          yum install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm && \
+          INSTALL_PKGS="python2-pip python-requests pytz" && \
           yum install -y --setopt=tsflags=nodocs $INSTALL_PKGS && \
           rpm -V $INSTALL_PKGS && \
-          yum clean all
+          yum clean all && \
+          pip install --no-cache-dir enum smartsheet-python-sdk
+          pip install urllib3 --upgrade && \
```

Comment:
Can we use the ``--no-cache-dir` as well when using the other `pip install` commands

Reply:
yes, makes sense - additional commit below to add '--no-cache-dir'